dialect/StablehloOps.td

```diff
@@ -0,0 +1,2362 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+   Copyright 2022 The StableHLO Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef STABLEHLO_DIALECT_STABLEHLO_OPS
+#define STABLEHLO_DIALECT_STABLEHLO_OPS
+
+include "dialect/Base.td"
+include "mlir/Dialect/Shape/IR/ShapeBase.td"
+include "mlir/IR/OpBase.td"
+include "mlir/Interfaces/InferTypeOpInterface.td"
+include "mlir/Interfaces/SideEffectInterfaces.td"
+include "mlir/IR/OpAsmInterface.td"
+
+def StableHLO_Dialect : Dialect {
+  let name = "stablehlo";
+  let cppNamespace = "::mlir::stablehlo";
+
```

Comment:
Consider adding a `let description = [{ ... }];` in case we plan to generate documentation from TD at some point.

Could use the repo description:


```suggestion
let description = [{
    StableHLO is an operation set that expresses ML computations. It has been originally 
    bootstrapped from the MHLO dialect and enhances it with additional functionality, including 
    serialization and versioning, to be used as a portability layer between ML frameworks 
    and ML compilers.
}];
```

Reply:
Thank you! That's a nice catch. We have a description for the StableHLO dialect in its current dummy state, but I accidentally deleted it when bootstrapping from MHLO. Will fix later today.